### PR TITLE
Removed the BrowserStack integration from ckeditor5-dev repository

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Removed the BrowserStack integration from `ckeditor5-dev` repository. See ckeditor/ckeditor5#1742.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/bin/test-travis.sh [hunk 1]
```diff
@@ -19,4 +19,4 @@ yarn run lint && \
 ${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies && \
 cd ${CKEDITOR5_TEST_ENVIRONMENT} && \
 node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --coverage --reporter=dots --browsers=Chrome && \
-node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --reporter=dots --browsers=Firefox,BrowserStack_Edge,BrowserStack_Safari
+node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --reporter=dots --browsers=Firefox
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/bin/test-travis.sh]: Or `Firefox,Chrome`?

Comment (ma2ciek) [reply packages/ckeditor5-dev-tests/bin/test-travis.sh]: And in the https://github.com/ckeditor/ckeditor5-react/pull/95/files we can have Chrome too I guess.

Comment (pomek) [reply packages/ckeditor5-dev-tests/bin/test-travis.sh]: No, we shouldn't. We want to generate CC report only once. If we merge those lines, we will generate two reports.

Comment (ma2ciek) [reply packages/ckeditor5-dev-tests/bin/test-travis.sh]: Ok, so that's fine for me.

**Pull request merged**